CODENAMES.md

```diff
@@ -11,5 +11,11 @@ releases are subject to change.
 * Hydrogen (18.x 2022)
 * Iron (20.x 2023)
 * Jod (22.x 2024)
+* Krypton (24.x 2025)
+* Lithium (26.x 2026)
+* Magnesium (28.x 2027)
+* Neon (30.x 2028)
```

Comment:
Alternatives:

- Neodymium
- Neptunium
- Nickel
- Nihonium
- Niobium
- Nitrogen
- Nobelium

Reply:
Nitrogen should be a reminder of "oxygen" previous release, so it would be cool, it should be used for relase with perf improvement to streamline the performance issue

Reply:
It can't be a reminder as it comes first 🙂.
FWIW Neon is a noble gas, similar to Argon and Krypton. It's also half as many characters (four instead of eight) as Nitrogen 🙂.